Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettings.cpp

```diff
@@ -46,7 +47,19 @@ namespace AzToolsFramework
                     ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
                     ->Attribute(AZ::Edit::Attributes::Max, 1.0f)
                     ->Attribute(AZ::Edit::Attributes::Step, 0.025f)
-                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &PaintBrushSettings::OnOpacityChange);
+                    ->Attribute(AZ::Edit::Attributes::ChangeNotify, &PaintBrushSettings::OnOpacityChange)
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::ComboBox, &PaintBrushSettings::m_blendMode, "Mode", "Blend mode of the paint brush.")
+                    ->EnumAttribute(PaintBrushBlendMode::Normal, "Normal")
+                    ->EnumAttribute(PaintBrushBlendMode::Multiply, "Multiply")
+                    ->EnumAttribute(PaintBrushBlendMode::Screen, "Screen")
+                    ->EnumAttribute(PaintBrushBlendMode::Add, "Linear Dodge (Add)")
+                    ->EnumAttribute(PaintBrushBlendMode::Subtract, "Subtract")
+                    ->EnumAttribute(PaintBrushBlendMode::Darken, "Darken (Min)")
+                    ->EnumAttribute(PaintBrushBlendMode::Lighten, "Lighten (Max)")
+                    ->EnumAttribute(PaintBrushBlendMode::Average, "Average")
+                    ->EnumAttribute(PaintBrushBlendMode::Overlay, "Overlay")
+                    ;
```

Comment:
If you switch to AZ_ENUM_CLASS_WITH_UNDERLYING_TYPE then the EnumAttribute calls can be reduced to:
->Attribute(AZ::Edit::Attributes::EnumValues, AZ::Edit::GetEnumConstantsFromTraits<PaintBrushBlendMode>())

It would use the exact enum name so you would have to rename them to whatever you want the user to see in the drop down.

Reply:
The problem is that the names that we want the user to see contain parentheses, so I don't believe GetEnumConstantsFromTraits() is a workable approach here.

Reply:
I understand. My suggestion implied removing the more descriptive naming or renaming the enums to include that wording. It could possibly replace parentheses with "_", if that's even necessary. The details for the more artist friendly names could also be moved to the property description tooltip or documentation. In any case, all of the comments were about sharing and simplifying code. Nothing blocking.